### ISSUE TITLE
Hero image is missing from specific wordpress blogs

### ISSUE THREAD

**Issue by avaida** [mozilla/readability#118]
**Reproducible on**:
Beta 38.0b2-build1 (20150406174117)

**Sample pages showing this issue**:
* http://delistraty.com/2015/03/16/phineas-gage-gauging-time/
* https://nerdybookclub.wordpress.com/2015/03/18/at-home-between-the-pages-by-dan-gemeinhart/

**Comment by n1k0** [comment]
Hero image markup:

```html
<div class="entry-thumbnail"> 
  <a href="http://delistraty.com/2015/03/16/phineas-gage-gauging-time/" title="Permalink to Phineas Gage, Gauging&nbsp;Time">
    <img width="570" height="353" src="https://lemanhattan.files.wordpress.com/2015/03/lead.jpg?w=570" class="attachment-post-thumbnail wp-post-image" alt="lead"></a>
</div>
```

**Comment by gijsk** [comment]
The first testcase is dead. What's the "hero image" for the second testcase? An image does get taken into account near the bottom of the page... is that the image you mean? Do you just expect it to be higher up?

**Comment by avaida** [comment]
Yeah, sorry about the second testcase, I was aiming for a different link when I filed this. Here are a few additional samples:
* http://delistraty.com/2014/11/26/the-loonies-and-lovelies-we-meet-while-traveling/
* http://delistraty.com/2014/06/02/the-ultimate-paris-to-do-list/
* http://delistraty.com/2014/03/04/a-messy-view-on-paris-aestheticism-and-reimagining-the-past/

**Comment by n1k0** [comment]
In these cases, the "hero image" is simply not part of the `entry-content` main div, which embeds the full text of the blog post. Not sure how we could find a way to detect and include this properly.

**Comment by n1k0** [comment]
Note that the full text of the blog post is detected and extracted just fine, which would make me marking this as trivial.

**Comment by gijsk** [comment]
> In these cases, the "hero image" is simply not part of the `entry-content` main div, which embeds the full text of the blog post. Not sure how we could find a way to detect and include this properly.

We could add things to the 'positive' class list regex so that the container of the post image (which has some wp-whatever class, presumably) gets scored, which might lead to a common container being uplifted. Maybe. But I agree this is a v2 thing. We could add a label for that? :-)
